cypress/fixtures/marriage/expected-marriage-record.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+const search = {
+  systemNumber: '',
+  forenames: 'Tester',
+  surname: 'SOLO',
+  day: '29',
```

Comment:
Would prefer the date to look like this

dom: {
  day: '29',
  month: '02',
  year: '2012'
}

and change subsequent usage accordingly.

Reply:
Done